baseem.js

```diff
@@ -0,0 +1 @@
+HI!

```

Comment:
Hey dude, the ticket you took said to add HI in lowercase.  Why are they in uppercase??